pixeltable/client.py

```diff
@@ -147,6 +145,50 @@ def create_table(
         _logger.info(f'Created table {path_str}')
         return tbl
 
+    def create_table_from_parquet(self, path_str: str, parquet_path: str,
```

Comment:
For now, the user for parquet import is not looking to import images. And the user for HF is just demos. 

These importer interfaces probably need to be reworked, beyond naming. Maybe you can suggest something.

1. One downside I see is: what happens if there is a new parquet file to 'append' to the data, how do we insert it?  we would also need an 'insert' method (in the table itself).   This lets you try things out before committing to a large dataset insert. Eg. in huggingface, the interface lets you  be selective about what slice of data you download (eg split or even individual file) and one maybe wants to insert more of the data later.

2. A second downside is how a user can specify the table schema to suit their needs to override default mappings, for example if a string type maps to an image url or is simply a string, or a type is not yet covered.

One option is we instead offer convenience functions to make schemas from different formats. pxt.create_schema_from_parquet(), which users can use with cl.create_table, and table `insert` functions. 

A separate discussion about cl and whats in it is probably due. Re. having many importers at `cl`,  it's  handy in getting users to discover stuff. Pandas does this, .read_csv(), .read_parquet(), .read_excel(), etc., as does sparksql, and other dataframe libraries, ray.data etc.  you can easily discover how to read your data and get started.  I think we want to weight that heavily here, and mimic this approach while handling any internal mess it may bring (with imports within functions, etc)

You raise a good question about `cl` as a class. Why not go for`pxt.create_table(), pxt.drop_table()`, with an implicit global client inited with `pxt.init()`. Then the user only needs to keep one name in mind: `pxt`, and their table/dataframes, instead of pxt, and cl. This approach would paralle interfaces of  pandas, ray, etc.  

Power users can use separate clients if they need to be explicit about different pixeltable clients in the same application, and pass these as arguments to those pxt.calls if needed.